app/controllers/api/base_controller/authentication.rb

```diff
@@ -212,13 +212,31 @@ def httpd_oidc_config_param(name)
       def oidc_provider_metadata
         @oidc_provider_metadata ||= begin
           oidc_provider_metadata_url = httpd_oidc_config_param("OIDCProviderMetadataURL")
+          return {} if oidc_provider_metadata_url.blank?
```

Comment:
I don't think this will set the memoization...prefer...

```ruby
@oidc_provider_metadata ||= begin
  if oidc_provider_metadata_url.blank?
    {}
  else
    uri = ...
    ...
    JSON.parse ...
  end
end
```